src/SOS/SOS.UnitTests/ConfigFiles/Windows/Debugger.Tests.Config.txt

```diff
@@ -164,6 +164,37 @@
             </Option>
           </Options>
         </Option>
+        <!--
+            DotnetDumpCommands (runs on every runtime except netcore2.1 on Linux)
+          -->
+        <Option>
```

Comment:
You shouldn't need to add this if you want to run across all the runtimes.  This is a dup basically of the above general prebuilt tests section on line 66.

Reply:
You haven't removed this section.  Are you?

Reply:
I thought you were talking about the comment...

Actually, I have to define a `TestName` variable because of my configuration for Linux right ?
Then, my unit tests method signature take into consideration the newly introduced TestName variable:

`[SkippableTheory, MemberData(nameof(GetConfigurations), "TestName", "DotnetDumpCommands")]`

In this case, if I don't add this block in Windows configuration file, there won't be any configuration matching `TestName=DotnetDumpCommands` predicate (on Windows environments).

I have the feeling that I'm in the exact same case as the `SOS.StackAndOtherTests` one (running for every .Net core versions + .Net framework one), for which every configuration is redefined as I did.

Did I misunderstand something ?

Reply:
I'm ok with leaving this now.